packages/webpack-cli/lib/groups/AdvancedGroup.js

```diff
@@ -57,6 +57,15 @@ class AdvancedGroup extends GroupHelper {
                 options.plugins = [hotModuleVal];
             }
         }
+        if (args.outputReporter) {
+            const { ReporterPlugin } = require('webpack');
+            const reporterVal = new ReporterPlugin();
```

Comment:
@rishabh3112 don't know why this is throwing the following error, need your help to debug -
 
![Screenshot from 2020-03-16 15-34-18](https://user-images.githubusercontent.com/46647141/76745317-ae4b8c80-679b-11ea-81a8-d717eaf8ed90.png)